### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by Enoisong**
### At this point in this project, I:
- Add some functionality to make my application  interactive
- Add a new JavaScript file and import it as a module:
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button.
- Store the updated array of items in local storage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,5 +139,5 @@ If you like this project, give me a star
 
 This project is [MIT](./MIT.md) licensed.
 
- <p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (BenMukebo) [inline README.md:142]: - [ ] Actually, you have two README files, the one from the root folder and the other in the src folder. Kindly remove one. 🚮

File: src/index.js [hunk 2]
```diff
@@ -20,3 +20,10 @@ addTodoBtn.addEventListener('click', () => {
     render(todosList);
   }
 });
+
+// clear all completed todos
+const clearBtn = document.querySelector('.clear-btn');
```
Comment (BenMukebo) [inline src/index.js:26]: - [ ] According to Project requirements you should create a new JavaScript file to contain methods related to the status updates (completed: true / false).and import it as a module:
- Kindly make this. 👎🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Enoisong,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻q
- No linter errors. ✔️ 
- GitHuvflow was correctly used.  ✔️   
- The task object's values for `completed` is updated upon user actions. ✔️   
- A function for the "Clear all completed" button is implemented. ✔️   
- The updated array of items is stored in local storage. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Enoisong,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

- [OPTIONAL] Kindly remove the Author on lines 98 and 99 in the Readme 

Screenshot 2023-04-07 at 19 08 08


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**